source/crud/tests/unified/deleteMany-let.yml

```diff
@@ -1,6 +1,6 @@
 description: "deleteMany-let"
 
-schemaVersion: "1.4"
+schemaVersion: "1.0"
```

Comment:
Oh, going back to my earlier question about why these had 1.4: I suppose this might have been copied this from `aggregate-let.yml`, which required it for `serverless: "forbid"`.

Reply:
Yes, this is what happened. I kept 1.4 in `aggregate-let.yml`